contracts/base/SemaphoreCore.sol

```diff
@@ -34,13 +34,14 @@ contract SemaphoreCore is ISemaphoreCore {
 
     uint256 signalHash = _hashSignal(signal);
 
-    return
-      verifier.verifyProof(
-        [proof[0], proof[1]],
-        [[proof[2], proof[3]], [proof[4], proof[5]]],
-        [proof[6], proof[7]],
-        [root, nullifierHash, signalHash, externalNullifier]
-      );
+    verifier.verifyProof(
+      [proof[0], proof[1]],
+      [[proof[2], proof[3]], [proof[4], proof[5]]],
+      [proof[6], proof[7]],
+      [root, nullifierHash, signalHash, externalNullifier]
+    );
+
+    return true; // TODO: This is redundant
```

Comment:
Can't we remove this line? Updating the extension contracts should be sufficient to make this change compatible

Reply:
Happy to do so! I left it to limit the scope of changes.

Reply:
Seems you need to run run `yarn prettier:fix` to pass tests.